htdocs/core/lib/functions2.lib.php

```diff
@@ -814,7 +814,7 @@ function get_next_value($db,$mask,$table,$field,$where='',$objsoc='',$date='',$m
     //print "masktri=".$masktri." maskcounter=".$maskcounter." maskraz=".$maskraz." maskoffset=".$maskoffset."<br>\n";
 
     // Define $sqlstring
-    $posnumstart=strrpos($maskwithnocode,$maskcounter);	// Pos of counter in final string (from 0 to ...)
+    $posnumstart=mb_strrpos($maskwithnocode,$maskcounter, 'UTF-8');	// Pos of counter in final string (from 0 to ...)
```

Comment:
Hi @rdoursenaud when I put Greek characters numbering reaches to 10.
With two ways properly run one is to add the mb_ * functions and second to activate the php.ini 
mbstring.internal_encoding = UTF-8
mbstring.func_overload = 7